BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.h

```diff
@@ -47,12 +47,12 @@ typedef void (^BOXSessionTaskReplacedBlock)(NSURLSessionTask *oldSessionTask, NS
  * An API call is considered to have succeeded if it returns with an HTTP status code in the
  * 2xx range, with an exception made for 202. 202, 3xx, 4xx, and 5xx are all treated as errors.
  *
- * NSURLConnectionDataDelegate
+ * BOXURLSessionTaskDelegate
  * ===========================
- * BOXAPIOperation instances issue API calls using NSURLConnection. Each BOXAPIOperation acts
- * as the delegate for its own API call. [BOXAPIOperations](BOXAPIOperation) are not concurrent
- * NSOperations, so they keep the runloop they are running on open during the lifetime of the
- * request to enable receiving delegate callbacks.
+ * BOXAPIOperation instances issue API calls using NSURLSession. Each BOXAPIOperation acts
```

Comment:
Since BOXAPIOperation is fairly high level, I think it makes sense to keep the abstraction terminology here. NSURLSession is as much as a class as it is a recognizable networking concept to iOS developers. Once we get into nitty gritty API calls, it makes more sense to use NSURLSessionTask if it's mentioned.